applications/DEMApplication/custom_strategies/strategies/explicit_solver_strategy.cpp

```diff
@@ -84,8 +88,7 @@ namespace Kratos {
                     break;
                 }
             }
-
-            if (found) continue;
+            if (found) return;
```

Comment:
Does this return end the function? We don't want this, but move to the next particle. This might be an error!

Reply:
If it's correct, looks scary or ambiguous.